src/app/_components/Header/HeaderButton.client.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import "@/app/_styles/studioPage.css";
```

Comment:
적용하신 버튼 css는 header.css로 옮기는 편이 낫겠네요! 아래 경로로 만들어놓겠습니다

studioPage.css 내부의 버튼 스타일링은 그대로 놔두었다가 리안님께서 적용하시면 그때 지울게요!

```suggestion
import "@/app/_styles/header.css";
```